native/helpers/challenges.js

```diff
@@ -450,6 +450,13 @@ export const challenges = [
     developer: 'sadanandpai',
     tags: [],
   },
+  {
+    title: 'Quiz App with Timer',
+    link: 'Quiz-app-with-timer',
```

Comment:
please do not use any capital letters

rename the folder as along

Reply:
Okay got it